createlinks

```diff
@@ -58,6 +58,18 @@ event_services($event, 'ejabberd' => 'restart');
 event_services($event, 'httpd' => 'reload');
 templates2events("/etc/ejabberd/ejabberd.yml",  $event);
 
+
+#
+# nethserver-sssd-save event
+#
+event_templates('nethserver-sssd-save',qw(
+    /etc/ejabberd/ejabberd.yml
+));
+
+event_services('nethserver-sssd-save', qw(
+    ejabberd restart
```

Comment:
Can we use `reload` instead?

Reply:
I just tested it again, and it seems that reload is a really tiny action

- modify parameters in the ldap ejabberd.yml file
- reload ejabberd
- pidgin is still able to login users
- restart ejabberd
- pidgin fails to connect

I think we should must restart